recipe/bld.bat

```diff
@@ -1,17 +1,16 @@
-mkdir build-conda
```

Comment:
this name was chosen on purpose to avoid conflicts when I'm working with local directory

Reply:
ok I will undo this modification!